Library/Homebrew/os/linux/elf.rb

```diff
@@ -101,7 +101,7 @@ def interpreter
   def patch!(interpreter: nil, rpath: nil)
     return if interpreter.blank? && rpath.blank?
 
-    if HOMEBREW_PATCHELF_RB_WRITE
+    if HOMEBREW_PATCHELF_RB_WRITE && ENV["HOMEBREW_NO_PATCHELF_RB_WRITE"].blank?
```

Comment:
Can remove this change if this is set out of process (which I think would be a bit less confusing).

Reply:
Done!